code/script-math.js

```diff
@@ -0,0 +1,56 @@
+var divisionPerformed = false;
+
+// displays numerous inputs (numbers) on the screen
+function displayNumber(inputValue) {
+  nameCalculator.nameDisplayMain.value = nameCalculator.nameDisplayMain.value + inputValue;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(inputValue) {
+  if (divisionPerformed == true && inputValue == 0) {
```

Comment:
I am not sure how it works in JS ... but I believe that you dont need compare ( == true ... is not needed)
when you create good name for property (e.g. .. isDivisionPerformed) .... so it should be enough ...
if (isDivisionPerformed && inputValue == 0)  ... test it

Reply:
TESTED and truly, it does work, good guess... changes gonna be visible once a commit them later on with more fixis